internal/pkg/acl/acl.go

```diff
@@ -205,6 +205,18 @@ func convertToFlags(operations ...any) string {
 	return strings.Join(ops, ", ")
 }
 
+func ConvertToLower(operations ...any) string {
+	var ops []string
```

Comment:
We already know the length of `ops`, so it's more efficient to allocate that space at the beginning.
```suggestion
	ops := make([]string, len(operations))
```